.idea/codeStyles/Project.xml

```diff
@@ -47,4 +47,4 @@
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
\ No newline at end of file
+</component>
```

Comment:
Unsure as to why this shows up. This file was untouched

Reply:
Does `git reset` have any effect?

Reply:
fixed this